src/network/networkctl.c

```diff
@@ -1518,6 +1518,25 @@ static int show_logs(const LinkInfo *info) {
                         NULL);
 }
 
+static int table_add_string_line(Table *table, const char *key, const char *value) {
```

Comment:
why not make this:

```c
static int table_add_string_line(Table *table, const char *key, TableDataType type, const void *value);
```

This is only minimally more verbose, but would allow you to use it for the TABLE_IN6_ADDR line (and similar) too

Reply:
(and one day we should add proper "vertical" mode to the table logic itself, with proper json generation and everything, i.e. where a specific column rather than the first line becomes the source of the json field names.)

Reply:
Because I'd like to suppress the line when `value` is an empty string. If it takes `type`, then it is hard to judge when we should suppress the line.